bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,241 @@
+          <li><img src="{{ static('img/firefox/os-new/icons/settings.png') }}" width="142" height="142" class="organized fade" alt="{{_('Settings')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/gallery.png') }}" width="142" height="142" class="entertained organized fade" alt="{{_('Gallery')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/usage.png') }}" width="142" height="142" class="organized fade" alt="{{_('Usage')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/marketplace.png') }}" width="142" height="142" class="connected entertained organized fade" alt="{{_('Marketplace')}}" /></li>
+        </ul>
+      </div>
+    </section>
+    <section class="fxos-marketplace">
+      <div class="content-wrapper">
+        <h2>{{_('Add more from the Firefox Marketplace')}}</h2>
+        <p>{{_('Get all the most popular games and social networking apps right here.')}}</p>
+        <ul class="apps-marketplace">
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/facebook.png') }}" width="142" height="142" alt="Facebook" />
+            <span>Facebook</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/youtube.png') }}" width="142" height="142" alt="YouTube" />
+            <span>YouTube</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/line.png') }}" width="142" height="142" alt="Line" />
+            <span>Line</span>
+          </li>
+        </ul>
+        <a href="//marketplace.firefox.com/" class="cta-button marketplace">{{_('Go to the Marketplace')}}</a>
+        <p class="footnote">{{_('* Pre-installed apps may vary by manufacturers and operators.')}}</p>
+      </div>
+    </section>
+
+    {% set feed_links = firefox_os_feed_links(LANG) %}
+    {% if feed_links %}
+    <section class="fxos-news">
+      <div class="content-wrapper">
+        <aside>
+          <h2>{{_('In the news')}}</h2>
+          <p><a href="">{{_('See more')}}</a></p>
+        </aside>
+        <ul>
+          {% for link, title in feed_links %}
+          <li><a href="{{ link }}">{{ title }}</a></li>
+          {% endfor %}
+        </ul>
+      </div>
+    </section>
+    {% endif %}
+
+    <section class="fxos-community">
+      <div class="content-wrapper">
+        <img src="{{ static('img/firefox/os-new/mozilla_groupshot.png') }}" width="260" height="260" alt="{{_('Mozilla community photo')}}" />
+        <h2>{{_('Powered by the Mozilla community')}}</h2>
+        <p>{{_('Mozilla is a non-profit organization fueled by a passionate global community. Together, we’re able to create and innovate with local interests in mind, while making the Web better and more accessible for everyone everywhere.')}}</p>
+        <p><a href="">{{_('Learn more about Mozilla')}}</a></p>
```

Comment:
I have actually asked where this should point as there are generally three possibilities:
mozilla.org
mozilla.org/contribute [seeing it is in a section focusing on community]
mozilla.org/mission

Not sure which ine yet, will follow up with Jen today.